apps/backend/src/modules/openai/libs/enums/openai-roles.ts

```diff
@@ -0,0 +1,7 @@
+const OpenAIRoles = {
+	ASSISTANT: "assistant",
+	SYSTEM: "system",
+	USER: "user",
+} as const;
+
```

Comment:
```suggestion
const OpenAIRole = {
	ASSISTANT: "assistant",
	SYSTEM: "system",
	USER: "user",
} as const;

```

https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md
B23

please update other places too